doc/api/index.rst

```diff
@@ -294,6 +294,7 @@ conversion of Python variables to GMT virtual files:
     clib.Session.virtualfile_from_grid
     clib.Session.virtualfile_in
     clib.Session.virtualfile_out
+    clib.Session.virtualfile_to_dataset
```

Comment:
At L287 above, could you change the sentence to read "These methods are context managers that automate the
conversion of Python variables to **and from** GMT virtual files"? Since we can convert GMT virtualfiles to Python objects now.

Reply:
Done in aee0499f7d01f81a35306c8133dc5c6cbba22615.